src/bot/commands/IgnoreCommand.ts

```diff
@@ -1,16 +1,26 @@
-import { Message, Permissions } from 'discord.js';
+import { Message } from 'discord.js';
 
 import * as ignoreList from '@util/db/IgnoreList';
 import localize from '@util/i18n/localize';
+import Config from '@config/Config';
 import Command from './base/Command';
+import userHasElevatedRole from './helpers/userHasElevatedRole';
 
 export default class IgnoreCommand implements Command {
   public readonly TRIGGERS = ['ignore'];
   public readonly USAGE = 'Usage: !ignore <user>';
 
+  private readonly config: Config;
+
+  constructor(config: Config) {
+    this.config = config;
+  }
+
```

Comment:
One last thing. Could you remove these unused config fields? Since you moved everything to `userHasElevatedRole`, all of these are not needed anymore :)

Reply:
Done! 😄